reflex/vars.py

```diff
@@ -1151,6 +1151,59 @@ def foreach(self, fn: Callable) -> Var:
             _var_is_local=self._var_is_local,
         )
 
+    def range(self, start: Var | int = 0) -> Var:
```

Comment:
Maybe instead of this we make a special function `rx.range` to match the Python API. Then we can have more flexibility